Tests/LibWeb/Layout/expected/abspos-flexbox-with-auto-height.txt

```diff
@@ -3,8 +3,7 @@ Viewport <#document> at (0,0) content-size 800x600 children: not-inline
     BlockContainer <body> at (10,10) content-size 780x0 children: not-inline
       Box <nav> at (11,11) content-size 310.453125x19 positioned flex-container(row) [FFC] children: not-inline
         BlockContainer <div.item> at (12,12) content-size 308.453125x17 flex-item [BFC] children: inline
-          line 0 width: 308.453125, height: 17, bottom: 17, baseline: 13.296875
-            frag 0 from TextNode start: 0, length: 36, rect: [12,12 308.453125x17]
+        frag 0 from TextNode start: 0, length: 36, rect: [12,12 308.453125x17] baseline: 13.296875
               "This should have a green background."
```

Comment:
Indentation seems wrong in these dumps. The fragment is still a child of `div.item` right?
And the text below the fragment is weirdly indented as well.

Reply:
oops, I removed too much indentation. now it should look right.